tools/create_certs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
A lot of tools pile up in the `tools` folder. What about creating a new folder `tools/certs` and move this script there. Then we can rename `tools/certs_configs` to `tools/certs/config`.